extensions/machine-learning-services/src/common/httpClient.ts

```diff
@@ -0,0 +1,57 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+import * as vscode from 'vscode';
+import * as fs from 'fs';
+import * as request from 'request';
+import * as constants from './constants';
+
+const DownloadTimeout = 20000;
+export class HttpClient {
```

Comment:
we don't have a module share with extension  right now. it can be registered as service

---
In reply to: [366617230](https://github.com/microsoft/azuredatastudio/pull/8870#discussion_r366617230) [](ancestors = 366617230)